scholia/app/templates/chemical.html

```diff
@@ -16,8 +16,18 @@
     }
   } LIMIT 500
 } AS %RESULTS {
-  INCLUDE %RESULTS
-  BIND(IRI(REPLACE(?formatterurl, '\\\\$1', str(?id))) AS ?idUrls).
+  { SELECT * WHERE {
+      INCLUDE %RESULTS
+      FILTER (?IDpred = wd:P234)
+      BIND(IRI(REPLACE(?formatterurl, '\\$1', REPLACE(REPLACE(?id, "\\+", '%2B'), '/', '%2F'))) AS ?idUrls).
```

Comment:
Would `ENCODE_FOR_URI` be better here? 

```SPARQL
SELECT * WHERE {
  BIND(ENCODE_FOR_URI("1S/C6H8O6/c7-1-2(8)5-3(9)4(10)6(11)12-5/h2,5,7-8,10-11H,1H2/t2-,5+/m0/s1") AS ?link)
}
```

Reply:
I'll test that today! Definitely better code :)

Reply:
Plz try again.